engineapi/backing_image_manager.go

```diff
@@ -12,8 +12,8 @@ import (
 const (
 	BackingImageManagerDefaultPort = 8000
 
-	CurrentBackingImageManagerAPIVersion = 2
-	MinBackingImageManagerAPIVersion     = 2
+	CurrentBackingImageManagerAPIVersion = 3
```

Comment:
Need to update the default backing image in the manifest as well. Also, create a release of https://github.com/longhorn/backing-image-manager and use that image as default. 

After that, update longhorn/longhorn as well.

Reply:
https://github.com/longhorn/backing-image-manager/releases/tag/v3_20220318